Dockerfile

```diff
@@ -1,12 +1,5 @@
-FROM debian:stretch-slim
+FROM scratch
 
 MAINTAINER CoreOS Inc. <coreos-dev@googlegroups.com>
 
 ADD bin/amd64/torcx /usr/sbin/torcx
```

Comment:
Thanks, I switched to COPY. I didn't add additional files, but added a meta-label pointing to this repository for sources.